src/app/plan/duck/operations.ts

```diff
@@ -242,23 +242,25 @@ const putPlan = planValues => {
       dispatch(Creators.updatePlan(latestPlan));
       const updatedMigPlan = updateMigPlanFromValues(latestPlan, planValues);
 
-      const fetch_retry: any = async n => {
-        for (let i = 0; i < n; i++) {
+      const fetchRetry: any = async totalTries => {
+        for (let currentTries = 0; currentTries < totalTries; currentTries++) {
           try {
             return await client.put(
               new MigResource(MigResourceKind.MigPlan, migMeta.namespace),
               latestPlan.metadata.name,
               updatedMigPlan
             );
           } catch (err) {
-            const isLastAttempt = i + 1 === n;
+            dispatch(commonOperations.alertErrorTimeout('Failed to update plan. Retrying...'));
```

Comment:
I'm not sure we want to spam them on retries, could be a relatively normal event and this might make it seems like something is badly wrong.